.github/workflows/roman_ci.yml

```diff
@@ -39,7 +40,7 @@ jobs:
       - id: path
         run: echo "path=${{ env.CRDS_PATH }}" >> $GITHUB_OUTPUT
       - id: server
-        run: echo "url=${{ env.CRDS_SERVER_URL }}" >> $GITHUB_OUTPUT
+        run: echo "url=${{ env.CRDS_LOCAL_URL }}" >> $GITHUB_OUTPUT
```

Comment:
I may be incorrect here (I don't fully understand CRDS), but GitHub Actions doesn't see the shared drive at `/grp/` and doesn't have permission to make a new directory at the root, so this may not accomplish the desired result of using the local cache, and I do not see how it will be able to retrieve CRDS files.